src/generated/client/.openapi-generator/FILES

```diff
@@ -1,3 +1,4 @@
+.openapi-generator-ignore
```

Comment:
As discussed previously, I believe all generated files can be removed from version control/ added to the .gitignore, This is what was requested for the app

Reply:
Done